whipper/common/mbngs.py

```diff
@@ -250,11 +325,19 @@ def _getMetadata(release, discid=None, country=None):
                     track.artist = trackCredit.getName()
                     track.sortName = trackCredit.getSortName()
                     track.mbidArtist = trackCredit.getIds()
+                    track.recordingArtist = recordingCredit.getName()
 
                     track.title = t.get('title', t['recording']['title'])
                     track.mbid = t['id']
                     track.mbidRecording = t['recording']['id']
-                    track.mbidWorks = _getWorks(t['recording'])
+                    works = _getWorks(t['recording'])
+                    track.mbidWorks = _getWorksId(works)
+                    track.composers = _getComposers(works)
+                    track.performers = []
+                    if 'artist-relation-list' in t['recording']:
```

Comment:
Why do you do this check here, when you don’t do one for `.composers`?

Reply:
I think that is handled in `_getComposers`.

Reply:
Exactly my point… ;)

Reply:
ahm so this should be resolved now ? (seems like i'm misunderstanding you)

Reply:
@ABCbum: I think @Freso is asking: why are you doing the check in `_getMetadata()` when the analogous one for the composers is in `_getComposers()`? So this still needs to be solved/answered.

Reply:
It's because the existing test cases are not fetched with needed `includes` then this way i won't have to replace all of them.

Reply:
It seems like not all recording have `artist-relation-list` https://musicbrainz.org/recording/69f6f250-e26f-4ce5-95a1-12129671ed7a for example

Reply:
The point is that this logic looks out of place here and that it could just as well be inside the `_getPerfomers()` function. Unless I’m missing something?

Reply:
And indeed, not all recordings have relationships. But this is also true for Works, so not something unique to Recordings—and thus not unique to the fetching of performers.

Reply:
Ahhh okay now i get your point finally.

Reply:
I have moved it to `_getPerformers`.